gprofiler/main.py

```diff
@@ -298,12 +314,19 @@ def verify_preconditions():
         sys.exit(1)
 
 
+def setup_signals() -> None:
+    # see my commit message for the reason we handle sigint manually.
```

Comment:
Better to copy it here.

Reply:
Fairly long. I'll add a shorter description + refer to the commit